quickstart/ruby/app.rb

```diff
@@ -0,0 +1,8 @@
+require 'sinatra'
+
+set :bind, '0.0.0.0'
```

Comment:
I think you're missing `set :port, 8080` given the default port Sinatra listens on is 4567.